src/sharedFramework/sharedFramework.proj

```diff
@@ -97,37 +100,25 @@
 
   <Target Name="GenerateRuntimeGraph">
     <PropertyGroup>
-      <runtimeGraphGeneratorName>DepsProcessor</runtimeGraphGeneratorName>
-      <runtimeGraphGeneratorProject>$(ProjectDir)tools-local/setuptools/independent/$(runtimeGraphGeneratorName)</runtimeGraphGeneratorProject>
-      <runtimeGraphGeneratorOutput>$(IntermediateOutputRootPath)setuptools/independent/$(runtimeGraphGeneratorName)</runtimeGraphGeneratorOutput>
-      <runtimeGraphGeneratorRuntime Condition="'$(OSGroup)'=='Windows_NT'">win</runtimeGraphGeneratorRuntime>
-      <runtimeGraphGeneratorRuntime Condition="'$(OSGroup)'=='OSX'">osx</runtimeGraphGeneratorRuntime>
-      <runtimeGraphGeneratorRuntime Condition="'$(runtimeGraphGeneratorRuntime)'==''">linux</runtimeGraphGeneratorRuntime>      
+      <SharedFrameworkAssetsFile>$(SharedFrameworkSourceRoot)/obj/project.assets.json</SharedFrameworkAssetsFile>
+      <SharedFrameworkDepsFile>$(SharedFrameworkNameAndVersionRoot)\$(SharedFrameworkName).deps.json</SharedFrameworkDepsFile>
     </PropertyGroup>
 
     <ItemGroup>
       <TrimPkgsFromDeps Include="runtime.$(PackageTargetRid).microsoft.netcore.dotnetapphost" />
       <TrimPkgsFromDeps Include="runtime.$(PackageTargetRid).microsoft.netcore.dotnethostresolver" />
       <TrimPkgsFromDeps Include="microsoft.netcore.dotnetapphost" />
       <TrimPkgsFromDeps Include="microsoft.netcore.dotnethostresolver" />
-      <TrimPkgsWithCmdPrefix Include="@(TrimPkgsFromDeps ->'--remove %(Identity)')" />
+    
+      <RuntimeGraphGeneratorRuntime Condition="'$(OSGroup)'=='Windows_NT'" Include="win" />
```

Comment:
Should this be a property instead?  Do we really want to allow multiple values?  I understand that DepsProcessor would allow you to pass multiple runtimes but we never really did that before.